src/CMakeLists.txt

```diff
@@ -153,10 +153,10 @@ endif()
 if(ONEDNN_BUILD_GRAPH)
     message(STATUS "Graph component is enabled")
 
-    if (NOT DNNL_GPU_RUNTIME STREQUAL "NONE" AND NOT DNNL_GPU_VENDOR STREQUAL "INTEL")
+    if (NOT DNNL_GPU_RUNTIME STREQUAL "NONE" AND DNNL_GPU_VENDOR STREQUAL "AMD")
```

Comment:
besides INTEL, NVIDIA, AMD, `DNNL_GPU_VENDOR` can also be NONE and GENERIC. Do we want to enable build for those?

Reply:
Good point. I didn't notice GENERIC vendor in https://oneapi-src.github.io/oneDNN/dev_guide_build_options.html.

Reply:
Fixed.